navigation/Navigation.tsx

```diff
@@ -44,34 +44,34 @@ export const Nav: React.FC<AppStackProps> = ({
     >
       <Group>
         <Screen name="Welcome" options={{ title: "Login to Strive" }}>
-          {(props: any) => <WelcomeScreens {...props} />}
+          {(props: any) => <WelcomeScreens {...props} firestoreCtrl={firestoreCtrl} />}
         </Screen>
         <Screen name="WelcomeConcept" options={{ title: "Final Screen" }}>
           {(props: any) => (
-            <WelcomeFinalScreen {...props} firestoreCtrl={{ firestoreCtrl }} />
+            <WelcomeFinalScreen {...props} firestoreCtrl={ firestoreCtrl } />
           )}
         </Screen>
         <Screen name="SignUp">
           {(props: any) => (
-            <SignUp {...props} firestoreCtrl={{ firestoreCtrl }} />
+            <SignUp {...props} firestoreCtrl={ firestoreCtrl } />
           )}
         </Screen>
         <Screen name="SignIn">
           {(props: any) => (
-            <SignInScreen {...props} firestoreCtrl={{ firestoreCtrl }} />
+            <SignInScreen {...props} firestoreCtrl={ firestoreCtrl } />
           )}
         </Screen>
         <Screen name="ForgotPassword">
           {(props: any) => (
             <ForgotPasswordScreen
               {...props}
-              firestoreCtrl={{ firestoreCtrl }}
+              firestoreCtrl={ firestoreCtrl }
             />
           )}
         </Screen>
         <Screen name="SetUp">
```

Comment:
Here the sign-up screen is searching for a navigation named SetUser when trying to set up an account in the only button attached to this screen. Resulting in an error and we can't go further than sign-up screen as a new user. Therefore we should modify the Screen name in this file to SetUser or modify the action made by the button to reflect this name. 

```suggestion
        <Screen name="SetUser">
```

Here's the error when trying to sign-up : 

```
(NOBRIDGE) ERROR  The action 'NAVIGATE' with payload {"name":"SetUser"} was not handled by any navigator.

Do you have a route named 'SetUser'?
```